src/main/java/org/corfudb/infrastructure/Rank.java

```diff
@@ -30,9 +30,16 @@
      */
     @Override
     public int compareTo(Rank other) {
+        /*
```

Comment:
please do not put such comments inside a commit PR and assume other team members will be diligent enough to remove this!

it is actually not a hack at all, it is a good safe solution, which appears in Lamport's original Paxos paper. But that's besides the point above.